packages/legend-query/src/components/QueryBuilderSubClassInfoTooltip.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type TooltipProps, Tooltip } from '@material-ui/core';
+import { StubTransition } from '@finos/legend-art';
+import { type Class, Multiplicity } from '@finos/legend-graph';
+import { getMultiplicityDescription } from './shared/QueryBuilderUtils';
+
+export const QueryBuilderSubClassInfoTooltip: React.FC<{
```

Comment:
Please note and change the spelling from `subClass` to `subclass`. Also I think this component is only ever used once in the explorer so let's not move it to a separate file and let's not export it.